### PR TITLE
Fixed Issue #452 and #421

### PR DESCRIPTION
**Pull request by deeyeet**
Expanding the descriptions also expands the course listings. Expanding the course listings also expands the descriptions. Added a space between CRNs and professor names.

Edit: Darkened the font of class times, CRNS, instructors and seat info from classes that are available. Rows that are conflicting still have lighter text.

Picture of updated text: https://imgur.com/0zMjNY3

### CODE REVIEW

**Comment by Bad-Science** [comment]
@deeyeet Thanks for the PR! Per our last discussion, I don't think clicking the course description should expand anything. Only the expand button should expand stuff.

File: web/src/app/listing/component.html [hunk 1]
```diff
@@ -42,7 +42,7 @@
 			<span *ngIf="showRemoveButton" (click)="removeButtonClick()" class="remove-button position-absolute"  [placement]="['right','bottom','left','top','auto']" ngbPopover="Click on this button to remove class from sidebar" container="body" triggers="mouseenter:mouseleave">
 				<img src="assets/images/remove.svg" />
 			</span>
-			<div class="menu position-absolute" (click)="showingMenu=!showingMenu">
+			<div class="menu position-absolute" (click)="showingMenu=!showingMenu" (click)="showingDescription=!showingDescription">
```
Comment (Bad-Science) [inline web/src/app/listing/component.html]: I think you should create an expand / toggle function to handle this rather than have two click handlers

**Comment by deeyeet** [comment]
Nevermind, the bug is still there.

File: web/src/app/listing/component.ts [hunk 2]
```diff
@@ -69,10 +72,27 @@ export class ListingComponent implements OnInit{
...
+    if (this.mouseMove) {
+      this.selectionService.toggleCourse(this.listing);
+    } 
+    
+    this.mouseMove = false;
+    this.mouseDown = false;
+
     // this.selectionService.toggleCourse(this.listing);
-    this.showingDescription= !(this.showingDescription);
+    // this.showingDescription= !(this.showingDescription);
```
Comment (Bad-Science) [inline web/src/app/listing/component.ts]: remove commented code

**Pull request merged**